CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+and will be the responsibility of the respective TrenchBoot maintainer(s).
+
+### TrenchBoot Original Repositories
+
+While the focus is on delivering capabilities into upstream projects, it is
+possible that a new or derivative code base may result to fulfill a role in a
+cross-cutting capability. The maintainer(s) for these code bases will be
+responsible for establishing coding styles and licensing that ensure compliance
+with all other TrenchBoot relevant projects.
+
+## Development
+
+We use github to host the project, to include tracking issues and feature
+requests, as well as accept pull requests.
+
+### Development Flow Overview
+
+Pull requests are the best way to propose changes to the project (we use
+[Github Flow](https://guides.github.com/introduction/flow/index.html)). We
+actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+### Contribution Licensing
+
+TrenchBoot is a cross-community integration project consisting of project
+repositories along with upstream project repositories. All contributions to a
+repository are made under the license that covers all work in that repository.
+
+### Issue Tracking
+
+Report issues, e.g. bugs, feature requests, etc., using Github's
+[issues](https://github.com/briandk/transcriptase-atom/issues)
+
+**Great Issues** tend to have:
+
+- A quick summary and/or background
+- For bug reports include:
+  - Steps to reproduce for bugs
+    - Be specific!
+    - Give sample code if you can.
+  - What you expected would happen
+  - What actually happens
+  - Notes (possibly including why you think this might be happening, or stuff
+    you tried that didn't work)
+- For feature requests include:
+  - An outline of how the feature would work
+  - Any dependecies the feature would require
+  - The benefit the feature will provide
+
+### Coding Style
+
+Markdown documents should be formatted to 80 columns to the extent possible.
+Exception to the 80 column is when column limitation breaks markdown rendering.
+
+Contributions targeting upstream project repositories will follow the upstream
+project's coding style rules.
+
+## License
+[![License: CC BY 4.0](https://i.creativecommons.org/l/by/4.0/88x31.png)
+](https://creativecommons.org/licenses/by/4.0/) This work is licensed under a
+[Creative Commons Attribution 4.0 International
+License](ttp://creativecommons.org/licenses/by/4.0/).  By contributing you
+agree that your contributions will be licensed under the Creative Commons
+Attribution 4.0 International License. Feel free to contact the maintainers if
+that's a concern.
+
+## Contacting Maintainers
+
+TrenchBoot is maintained by:
+ - Daniel P. Smith <dpsmith@apertussolutions.com>
```

Comment:
Probably should reword this to be,

TrenchBoot is maintained by TrenchBoot Steering Committee, current members:
Daniel P. Smith
Ross Philipson
Piotr Krol
Rich Persaud